homeassistant/components/switcher_kis/light.py

```diff
@@ -127,3 +135,20 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         await self._async_call_api(API_SET_LIGHT, DeviceState.OFF, self._light_id)
         self.control_result = False
         self.async_write_ha_state()
+
+
+class SwitcherSingleLightEntity(SwitcherBaseLightEntity):
+    """Representation of a Switcher single light entity."""
+
+
+class SwitcherDualLightEntity(SwitcherBaseLightEntity):
+    """Representation of a Switcher dual light entity."""
+
+    @property
+    def is_on(self) -> bool:
+        """Return True if entity is on."""
+        if self.control_result is not None:
+            return self.control_result
+
```

Comment:
```suggestion
        """Return True if entity is on."""
          super().is_on()
```

Reply:
It is done.

Reply:
This was just an example not something you should commit (should be something like `return `super().is_on() or ...`, however looking at the final result I think it would be better to remove this from the base class and add it in the subclasses.

Reply:
It is done.